api/v1alpha1/tritoninterfaceserver_types.go

```diff
@@ -42,6 +42,11 @@ type TritonInterfaceServerSpec struct {
 
 	// +kubebuilder:validation:Optional
 	PodResources Resource `json:"podResources"`
+
+	// TLSSpec includes the TLS Secret Name (optional)
```

Comment:
optional/required is defined by kubebuilder labels and docs processes it. So no need to mention again in the description